test/integration/routes_test.rb

```diff
@@ -96,5 +96,9 @@ class RoutesTest < ActionDispatch::IntegrationTest
   test "test get request for spectrums compare_search" do
     assert_routing({ path: '/spectrums/compare_search', method: 'get' }, {controller: 'spectrums', action: 'show', id: 'compare_search'})
   end
+  
+  test "test get request for spectrums set_search" do
+   assert_routing({ path: '/spectrums/set_search', method: 'get' }, {controller: 'spectrums', action: 'show', id: 'set_search'})
```

Comment:
```suggestion
   assert_routing({ path: '/spectrums/set_search', method: 'get' }, {controller: 'spectrums', action: 'set_search', id: 'set_search'})
```
@anantraghuvanshi The action should be `set_search`

Reply:
Nevermind, the test would fail if the action was changed back to 'set_search'.